### DISCUSSION TITLE
Code formatting with black

### DISCUSSION THREAD

**Discussion by Blucknote** [seruva19/kubin#81] [General]
Can i suggest [black](https://github.com/psf/black) as project code formatting tool?
It allow do not change coding habit, just set desire line width and black does the rest.

Black have multiple IDE extensions

**Comment by seruva19** [comment]
I will try it.

**Comment by seruva19** [comment]
I tried it. Looks pretty clean. Breaks some of my coding habits, but it's OK 😅 So, I will use it.
